src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityCreateController.php

```diff
@@ -38,6 +45,28 @@ class EntityCreateController extends Controller
 {
     use EntityControllerTrait;
 
+    /**
+     * @var ProtocolChoiceFactory
+     */
+    private $protocolChoiceFactory;
+
+    public function __construct(
```

Comment:
I think it's getting a bit messy now with the EntityControllerTrait and the overloaded constructor.

Reply:
Yes, I'm with you on this one. We could reconsider the trait solution in a future development effort. For now, I'd like to propose to not change this at this point.

Reply:
I'm fine with that